src/client/eppo-client.ts

```diff
@@ -988,6 +1000,32 @@ export default class EppoClient {
     validateNotBlank(flagKey, 'Invalid argument: flagKey cannot be blank');
 
     const flagEvaluationDetailsBuilder = this.newFlagEvaluationDetailsBuilder(flagKey);
+    const overrideVariation = this.overridesStore?.get(flagKey);
+    if (overrideVariation) {
+      const overrideAllocationKey = 'override-' + overrideVariation.key;
+      const flagEvaluationDetails = flagEvaluationDetailsBuilder
+        .setMatch(
+          0,
+          overrideVariation,
+          { key: overrideAllocationKey, splits: [], doLog: false },
+          null,
+          undefined,
+        )
+        .build('MATCH', 'Flag override applied');
+
+      return {
+        flagKey,
+        subjectKey,
+        variation: overrideVariation,
+        subjectAttributes,
+        flagEvaluationDetails,
+        doLog: false,
+        format: '',
+        allocationKey: 'override',
+        extraLogging: {},
+      };
```

Comment:
Although its a style thing: consider returning this response in a helper method in-line with other responses in this file